docs/source/tutorials/configuration.ipynb

```diff
@@ -1,20 +1,452 @@
 {
```

Comment:
<ul><li>the ID for whcih they want to overraide some   -&gt;  the ID for which they want to override some </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>